tcf_website/tests/test_instructor_edgecases.py

```diff
@@ -0,0 +1,161 @@
+# pylint: disable=no-member
+"""Tests for Instructor model with edge cases."""
+
+from django.test import TestCase
+
+from .test_utils import setup
+
+
+class InstructorEdgeTestCase(TestCase):
+    """Tests for instructor model with edge cases."""
+
+    def setUp(self):
+        setup(self)
+
+    def test_average_rating_for_course_no_reviews(self):
+        """Test average rating for a particular course when there are no reviews."""
+
+        self.review1.delete()
+        self.review2.delete()
```

Comment:
The thing is there are other reviews created used for other tests and although that one-liner would not interrupt anything, it might be better practice to show the specific reviews that are part of that course.